tf/examples/Bonsai/bonsai_example1.py

```diff
@@ -0,0 +1,110 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
How does `bonsai_example` and `bonsai_example1` differ ? Do we need a separate file ?